packages/atlas/src/providers/transactionManager/useTransaction.ts

```diff
@@ -30,13 +32,19 @@ type HandleTransactionFn = <T extends ExtrinsicResult>(opts: HandleTransactionOp
 const TX_SIGN_CANCELLED_SNACKBAR_TIMEOUT = 7000
 const TX_COMPLETED_SNACKBAR_TIMEOUT = 5000
 const MINIMIZED_SIGN_CANCELLED_SNACKBAR_TIMEOUT = 7000
+const RETRIES_TIMEOUT = 1000
+const RETRIES = 10
 
 export const useTransaction = (): HandleTransactionFn => {
   const { addBlockAction, setDialogStep, setErrorCode, addPendingSign, removePendingSign } = useTransactionManagerStore(
     (state) => state.actions
   )
   const nodeConnectionStatus = useConnectionStatusStore((state) => state.nodeConnectionStatus)
   const { displaySnackbar } = useSnackbar()
+  const [getTransactionStatus, { refetch }] = useGetMetaprotocolTransactionStatusEventsLazyQuery({
```

Comment:
Can we rename this `refetch` to something more descriptive, maybe `refetchTransactionStatus`?